### PR TITLE
Adding L1 & L2 Losses and respective tests

### PR DESCRIPTION
**Pull request by Shashi456**
Builds and passes locally.
Reference #127
Sorry for the onslaught of additions today, hope it's alright :P.

### CODE REVIEW

File: Sources/TensorFlow/Loss.swift [hunk 1]
```diff
@@ -12,6 +12,30 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/// Computes the L1 loss between predictions and labels.
```
Comment (rxwei) [inline Sources/TensorFlow/Loss.swift]:
```suggestion
/// Returns the L1 loss between predictions and labels.
```

File: Sources/TensorFlow/Loss.swift [hunk 2]
```diff
@@ -12,6 +12,30 @@
...
+///   - predicted: Predicted outputs from a neural network.
+///   - labels: Expected values, i.e. targets, that correspond to the correct output.
+@differentiable(wrt: predicted)
+public func l1Loss<Scalar: TensorFlowFloatingPoint>(
+    predicted: Tensor<Scalar>, expected: Tensor<Scalar>
+) -> Tensor<Scalar> {
+    return abs(expected - predicted).sum()
+}
+
+/// Computes the L2 loss between predictions and labels.
```
Comment (rxwei) [inline Sources/TensorFlow/Loss.swift]:
```suggestion
/// Returns the L2 loss between predictions and labels.
```

File: Sources/TensorFlow/Loss.swift (hunk 1 above)
Comment (rxwei) [reply Sources/TensorFlow/Loss.swift]: (#160 for context)

File: Tests/TensorFlowTests/LossTests.swift [hunk 3]
```diff
@@ -16,6 +16,23 @@ import XCTest
 @testable import TensorFlow
 
 final class LossTests: XCTestCase {
+
```
Comment (rxwei) [inline Tests/TensorFlowTests/LossTests.swift]: Remove extra line.

Comment (Shashi456) [reply Tests/TensorFlowTests/LossTests.swift]: @rxwei I'm sorry for these weird popups in my commits, I've found out that git has this nasty habit of trying to fix line endings and adding extra lines after functions start.

**Comment by Shashi456** [comment]
Also @rxwei when is swift-apis v4 slotted? o.O I have no idea.

**Comment by rxwei** [comment]
We've decided to delay v0.4 until a couple more fixes and library cleanups.

**Comment by Shashi456** [comment]
@rxwei could you trigger a build on this

**Pull request merged**